package/libs/zlib/patches/003-attach-sourcefiles-in-patch-002-to-buildsystem.patch

```diff
@@ -0,0 +1,100 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8e75f66..24d7329 100644
```

Comment:
could you add the source of this patch as a comment please.

Reply:
There's no upstream source, I created it myself.